301redirects.json

```diff
@@ -1645,5 +1645,9 @@
   "/docs/runtime_suite/auth0-client/overview_and_usage": {
     "destination": "/docs/runtime_suite/auth0-client/overview",
     "addedOn": "2024-01-22"
+  },
+  "/docs/console/company-configuration/providers/extensions/orchestrator-generator": {
```

Comment:
couldn't you leave the id as it was before, to avoid the redirect?

Reply:
per non cambiare l'id avrei dovuto tenere entrambi i file dentro a `/extensions`, non mi dispiaceva l'idea di raggruppare i file